### PR TITLE
Adds gopacket project

### PR DESCRIPTION
**Pull request by catenacyber**
Another golang project from google

### CODE REVIEW

File: projects/gopacket/fuzz_layer.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Catena cyber All rights reserved.
```
Comment (inferno-chromium) [inline projects/gopacket/fuzz_layer.go]: Please fix this.

File: projects/gopacket/project.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+homepage: "https://github.com/google/gopacket"
+primary_contact: "p.antoine@catenacyber.fr"
```
Comment (inferno-chromium) [inline projects/gopacket/project.yaml]: Need to get approval from maintainers and then use them here as primary contact. move yourself to auto_ccs field.

File: projects/gopacket/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+#TODO change
+MAINTAINER p.antoine@catenacyber.fr
```
Comment (inferno-chromium) [inline projects/gopacket/Dockerfile]: This TODO and maintainer field is not needed, please remove.

File: projects/gopacket/fuzz_layer.go (hunk 1 above)
Comment (Dor1s) [inline projects/gopacket/fuzz_layer.go]: I don't think we can check in code with such copyright :/ Please use Google copyright or move this file to upstream repo (preferred)

File: projects/gopacket/project.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (Dor1s) [inline projects/gopacket/project.yaml]: Can you also CC project maintainer(s) on this PR? We need them to review the fuzz target + someone else has to be the primary contact / auto_ccs

File: projects/gopacket/Dockerfile (hunk 3 above)
Comment (catenacyber) [reply projects/gopacket/Dockerfile]: ok

File: projects/gopacket/fuzz_layer.go (hunk 1 above)
Comment (catenacyber) [reply projects/gopacket/fuzz_layer.go]: I will do a PR to the upstream repo as suggested by @Dor1s

File: projects/gopacket/project.yaml (hunk 2 above)
Comment (catenacyber) [reply projects/gopacket/project.yaml]: Ok

File: projects/gopacket/fuzz_layer.go (hunk 1 above)
Comment (catenacyber) [reply projects/gopacket/fuzz_layer.go]: Ok, will try the preferred way

File: projects/gopacket/project.yaml (hunk 4 above)
Comment (catenacyber) [reply projects/gopacket/project.yaml]: I will do a PR for the fuzz target and see who answers

**Comment by Dor1s** [comment]
Ping :)

**Comment by catenacyber** [comment]
Waiting for https://github.com/google/gopacket/pull/737

**Comment by tomfitzhenry** [comment]
> Waiting for google/gopacket#737

This is now unblocked.

**Comment by inferno-chromium** [comment]
@catenacyber - can you please start a new PR with go-fuzz (not go-fuzz-build).

**Comment by gconnell** [comment]
LGTM from project maintainer (gopacket)

**Comment by catenacyber** [comment]
New PR opened

**Pull request closed without merging**